selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/CommitFilesByMultiSelectTest.java

```diff
@@ -181,7 +181,6 @@ public void commitFilesByMultiSelect() {
     git.closeGitInfoPanel();
     projectExplorer.selectMultiFilesByCtrlKeys(
         PROJECT_NAME + "/my-webapp/src/main/java/helloworld/AppController.java");
-    menu.runCommand(TestMenuCommandsConstants.Git.GIT, TestMenuCommandsConstants.Git.ADD_TO_INDEX);
```

Comment:
Does it really works without adding changes to index?

Reply:
Yes, of course. We can commit changes without add to index from UI Git command

Reply:
But in that case it doesn't make sense to edit the files `GreetingController.java`, ...